docs/sidebars.js

```diff
@@ -51,6 +51,19 @@ module.exports = {
       "codes/power",
       "codes/keymap-upgrader",
     ],
+    Configuration: [
+      "config/index",
+      "config/backlight",
+      "config/behaviors",
+      "config/combos",
+      "config/displays",
+      "config/encoders",
+      "config/keymap",
+      "config/kscan",
+      "config/power",
+      "config/system",
+      "config/underglow",
```

Comment:
These render as System and RGB Underglow rather than System and Underglow, so maybe they should be flipped for alphabetical order? Or RGB could be removed from the sidebar title.